lib/perl/Genome/Model/ClinSeq/Command/UpdateAnalysis.pm

```diff
@@ -194,6 +194,15 @@ class Genome::Model::ClinSeq::Command::UpdateAnalysis {
               is => 'Boolean',
               doc => 'Allow imported instrument data to be included',
         },
+        validation_as_exome => {
+              is => 'Boolean',
+              doc => 'Treat validation data as exome data. Includes instrument data from validation capture in exome reference alignment model.',
+        },
+        my_trsn => {
+              is => 'Text',
+              doc => 'If using validation-as-exome option, option to specify what instrument data to derive the target-set-region-name and region-of-interest from. valid options: exome or validation', 
+              valid_values => ['exome','validation']
+        },
```

Comment:
Would it be better to combine `validation_as_exome` and `my_trsn` as one parameter with the option of null, exome or validation. That way the user can never call `my_trsn` without enabling the `validation_as_exome` option. @acoffman  is it possible in UR to accept valid values of nothing and something (exome, validation)? Also I think that the name 'validation as exome' is a little confusing, perhaps `add_validation`?